tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>I am still in favor of renaming this "calculate_auc" to "predict" although it is not really a prediction. But maybe I just still get weirded out that someone uses an evaluation metrics as the actual prediction ^_^</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Hahah yeah we can find something else, especially since the AUC is not used in the case of regression. predict sounds good :)